src/Components/CreatePlayList.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from "react";
+import { useData } from "../Context/DataProvider";
+import { checkIfVideoIsAddedToPlaylist } from "../utils";
+export function CreatePlayList() {
+  const [playlistName, setPlaylistName] = useState("");
+  const [createPlaylist, setCreatePlaylist] = useState(false);
+  const {
+    state: { playlist, videoId },
+    dispatch
+  } = useData();
+  function createPlayList() {
+    setPlaylistName("");
+    dispatch({ type: "CREATE_PLAYLIST", payload: playlistName });
+    setCreatePlaylist(false);
+  }
+  function addToPlayList(playlistId) {
+    dispatch({ type: "TOGGLE_ADD_TO_PLAYLIST", payload: playlistId });
+  }
+  return (
+    <div>
+      <div className="padding-bottom">Save to..</div>
+      <hr />
+      <div className="padding-top padding-bottom">
+        {playlist.length > 0 ? (
+          <div>
+            {playlist.map(({ playlistName, playlistId, playlistVideos }) => (
+              <div key={playlistId}>
+                <label>
+                  <input
+                    type="checkbox"
+                    checked={checkIfVideoIsAddedToPlaylist(
+                      playlistVideos,
+                      videoId
+                    )}
+                    onChange={() => addToPlayList(playlistId)}
```

Comment:
Please take a look at here as well. Refactor it as per naming conventions.

Reply:
I have done it.